README.mediawiki

```diff
@@ -514,6 +514,11 @@ The following flavors of Linux are supported
 
 == Changes ==
 
+;2.2.0
+* Extend FileSystem and HardDrive to ZenPackLib
```

Comment:
The changes section should be for user-facing documentation of changes from the previous version. Implementation details like converting classes to zenpacklib isn't relevant to the user.

Same for the "Clean up extraneous non-ZPL style artifacts" below.